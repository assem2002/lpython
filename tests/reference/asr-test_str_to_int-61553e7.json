{
    "basename": "asr-test_str_to_int-61553e7",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_str_to_int.py",
    "infile_hash": "da0ed82ada98ef2757178c4997af53abc82f4761a0e683a613216c64",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_str_to_int-61553e7.stderr",
    "stderr_hash": "18dc06e9636f1028811bf8add4615471ff55ced13de0bb9c6c9c35c6",
    "returncode": 2
}