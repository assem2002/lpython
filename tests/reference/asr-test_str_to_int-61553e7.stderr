semantic error: invalid literal for int() with base 10: '3abc'
 --> tests/errors/test_str_to_int.py:2:15
  |
2 |     print(int('3abc'))
  |               ^^^^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
